tox_core/src/dht/server/mod.rs

```diff
@@ -1373,9 +1376,9 @@ mod tests {
 
     use std::net::SocketAddr;
 
-    const ONION_RETURN_1_PAYLOAD_SIZE: usize = ONION_RETURN_1_SIZE - secretbox::NONCEBYTES;
-    const ONION_RETURN_2_PAYLOAD_SIZE: usize = ONION_RETURN_2_SIZE - secretbox::NONCEBYTES;
-    const ONION_RETURN_3_PAYLOAD_SIZE: usize = ONION_RETURN_3_SIZE - secretbox::NONCEBYTES;
+    const ONION_RETURN_1_PAYLOAD_SIZE: usize = ONION_RETURN_1_SIZE - xsalsa20poly1305::NONCE_SIZE;
```

Comment:
We got these consts in tox_core/src/dht/codec.rs. Why did we duplicate them?

Reply:
They are defined in tests only, and private. Can be shared, but then need to find ath appropriate place for them.